requirements.in

```diff
@@ -0,0 +1,13 @@
+elasticsearch==7.9.1
+faiss_cpu
```

Comment:
probably we need to define the version of this library

Reply:
Done